guides/common/modules/proc_synchronizing-ubuntu-expanded-security-maintenance-content.adoc

```diff
@@ -0,0 +1,78 @@
+[id="Synchronizing-Ubuntu-Expanded-Security-Maintenance-content_{context}"]
+= Synchronizing Ubuntu Expanded Security Maintenance content
+
+Canonical provides Ubuntu https://ubuntu.com/security/esm[Expanded Security Maintenance] (ESM) repositories to their paying customers.
+Use {Project} to synchronize ESM repositories from Canonical to distribute content to your hosts running Ubuntu 14.04 and Ubuntu 16.04.
+
+Extract the ESM repository information from an existing host running Ubuntu 14.04 or 16.04 once by hand to enable synchronization to {Project}.
+
+[IMPORTANT]
+====
+Providing https://ubuntu.com/security/esm[Ubuntu ESM] repositories to hosts running Ubuntu 14.04 or 16.04 requires a subscription from Canonical.
+Ensure to provide valid licenses for all used Canonical products.
+Using insufficient, invalid, or otherwise inadequate licenses might violate your terms with Canonical.
+====
+
+.One time procedure on an existing host
+. Install `ubuntu-advantage-tools` on a host running Ubuntu 14.04 or 16.04:
++
+[source,none]
+----
+# apt-get install ubuntu-advantage-tools
+----
+. Register your host with Canonical:
++
+[source,none]
+----
+# ua attach ABCDEF0123456789
```

Comment:
```suggestion
# ua attach _ABCDEF0123456789_
```
User-entered string. This might require different settings for the code block.

Reply:
Fixed; changed for all code blocks.